docs/running-on-hardware.md

```diff
@@ -4,81 +4,15 @@ In order to run on hardware, the deployment needs to be built for the given hard
 
 ## Cross-Compiling for Embedded Arm Linux
 
-In the [environment setup](./prerequisites.md) for this tutorial, the ARM Linux cross-compilers were installed. In this section, we will use these compilers to cross-compile for ARM Linux by utilizing integration files called "CMake Toolchain". F´ provides cross-compiler integration for standard ARM Linux cross-compilers (i.e. those provided as binary downloads from ARM).
+In the [environment setup](./prerequisites.md) for this tutorial, the ARM Linux cross-compilers were installed. In this section, we will use these compilers to cross-compile for ARM Linux by utilizing integration files called "CMake Toolchain". Follow the steps in [Section 2](https://github.com/nasa/fprime/blob/devel/docs/Tutorials/CrossCompilationSetup/CrossCompilationTutorial.md#2-compiling-for-arm) of the Cross Compilation Tutorial for steps on how to compile for ARM.
 
-> Now is a good time to ensure that the ARM toolchains were installed properly. To test, run the following command:
-> ```shell
-> # For in-person workhops and 64-bit ARM hardware
-> /opt/toolchains/bin/aarch64-none-linux-gnu-gcc -v
-> 
-> # For 32-bit ARM hardware
-> /opt/toolchains/bin/arm-none-linux-gnueabihf-gcc -v
-> ```
->
-> Any output other than "file/command not found" is good.
-> 
-> macOS users must run these commands from within the Docker container described in [Appendix I](./appendix-1.md). Ensure this [script](https://github.com/fprime-community/fprime-workshop-led-blinker/main/bin/macos-docker) was downloaded to `led-blinker/bin/macos-docker` and run it in the terminal.
-
-Now cross-compiling is as easy as building the deployment for a specific platform. For users running on 64-bit arm the platform is called `aarch64-linux`, and for users on 32-bit linux use `arm-hf-linux`. This package expects the environment variable `ARM_TOOLS_PATH` to point to the installation directory of the ARM cross-compilers.
-
-> Users need to generate for each platform they wish to run on.  We previously generated for our host machine.
-
-Here is how to build for the 64-bit Arm Linux platform:
-
-```shell
-# In led-blinker/LedBlinker
-export ARM_TOOLS_PATH=/opt/toolchains
-
-# For in-person workshops and ARM 64-bit hardware
-fprime-util generate aarch64-linux
-fprime-util build aarch64-linux
-
-# For ARM 32-bit hardware
-fprime-util generate arm-hf-linux
-fprime-util build arm-hf-linux
-```
-> macOS users must run from within the Docker container in the directory `/project/LedBlinker`
+> During the step for running the `fprime-util generate` and `fprime-util build` commands, macOS users must run from within the Docker container in the directory `/project/LedBlinker`
```

Comment:
I mentioned this just in case the user didn't navigate to /project/LedBlinker already